docs/building-apps/moneygram-access-integration-guide.mdx

```diff
@@ -310,10 +422,57 @@ Code for submitting transactions to Stellar should be developed thoughtfully. Th
 - Set a maximum timebound on the transaction. This ensures that if your transaction is not included in a ledger before the set time, you can reconstruct the transaction with a higher offered fee and submit it again with better chances of inclusion.
 - Resubmit the transaction when you get 504 status codes. 504 status codes are just telling you that your transaction is still pending -- not that it has been canceled or that your request was invalid. You should simply make the request again with the same transaction to get a final status (either included or expired).
 
-Below is highly simplified code for submitting a payment transaction. It does not use timebounds, handle a transaction’s expiration, or handle 504 status codes.
+Below is a sample `Typescript` code for submitting a payment ("transfer") transaction using the Wallet SDK. It uses timebounds and handles 504 status codes (inside `submitTransaction`), but it does not handle a transaction’s expiration.
+
+Please note that the `Python` code is highly simplified. It does not use timebounds, handle 504 status codes or handle a transaction’s expiration.
 
 <CodeExample>
 
+```ts
+import { Wallet, IssuedAssetId } from "@stellar/typescript-wallet-sdk";
+
+const wallet = Wallet.TestNet();
+
+// This creates a Stellar instance to manage the connection with Horizon.
+const stellar = wallet.stellar();
+
+// Creates a stellar asset, in this case it should use USDC configuration.
+const asset = new IssuedAssetId(ASSET_CODE, ASSET_ISSUER);
```

Comment:
Let's define on how to query asset above (see another comment)

Reply:
https://github.com/stellar/stellar-docs/pull/658/commits/295515b170ec57706bff7d542ac9e2a0398b0334